File: clox/hello.c
#include <stdio.h>
#include "a.h"

int main(int argc, char** argv) {
    printf("Hello, world!\n");
    sayA();

    for (int i = 0; i < argc; i++) {
        printf("arg %d: %s\n", i, argv[i]);
    }

    return 0;
}

File: jlox/hello.java
package jlox;
import jlox.A;
import jlox.B;

class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello, world!");
        A.sayA(); // because its static
        B b= new B(); // because it's not static
        b.sayB();

        for (int i = 0; i < args.length; i++) {
            System.out.println("arg " + i + ": " + args[i]);
        }
    }
}
